### ISSUE TITLE
App crashing on selection of gif(Android)

### ISSUE THREAD

**Issue by Aman1706** [Giphy/giphy-react-native-sdk#90]
## 🐛 Bug Report

If a user tries to select a gif in the GiphyDialog menu, the app crashes 
This is happening for android and only for an apk or aab build, it works fine when running in debug.

I was using an older version of this package and then bumped it up to the latest, after that this issue occurred.

Looking at sentry I see this error 

```
RuntimeException com.android.internal.os.RuntimeInit$MethodAndArgsCaller in run
New Issue
Unhandled
java.lang.reflect.InvocationTargetException

```
## To Reproduce

(Write your steps here:)

1. Open the Gif menu on an android device 
1. Select a gif

## Your Environment

- Giphy React Native SDK version: 2.1.0
- React Native version:0.68.5
- Platform(s) (iOS, Android, or both?): android 
- Device info (Simulator/Device? OS version? Debug/Release?): Pixel 3A ARM64 emulator - Android 12 (Bug happens for release build only)

## Reproducible Demo

https://user-images.githubusercontent.com/41548441/207300939-59059877-fbd2-401d-9429-3eedfa320b68.mov

**Comment by Aman1706** [comment]
FYI I downgraded the version to 2.0.0 and it works fine there 
Looks like its some issue with 2.1.0

**Comment by pshoniuk** [comment]
Hi @Aman1706! Sorry for the delayed response. We can't reproduce the issue with an example app. Could you please create some minimal playground where this problem occurs? Or, if you have more detailed information about the error, such as a stake trace, could you please share it? That would be very helpful

**Comment by ujjwal-techahead-personal** [comment]
I too am facing the same issue. Only happens in release builds on Android.

**Comment by pshoniuk** [comment]
Hi @ujjwal-techahead-personal
Thank you for the report. We reproduced the issue on our side. The problem seems to be related to ProGuard misconfiguration. I'll update the status here when we have new information.

**Comment by pshoniuk** [comment]
@ujjwal-techahead-personal We've released a fix in [v2.1.3](https://github.com/Giphy/giphy-react-native-sdk/releases/tag/v2.1.3). Please check it when you get a chance. If the problem still occurs, please reopen this issue or create a new one
